### DISCUSSION TITLE
Am I replying correctly?

### DISCUSSION THREAD

**Discussion by V8Qxdvixstjg3n7afA** [simple-login/app#456] [Q&A]
I have recently noticed that, when I check sender emails in ProtonMail, they appear to be SimpleLogin email addresses. As such, I want to make sure that I am replying in a way that doesn't expose my email address. I used to copy the sender's email address and then generate the proper response email through the "send email" feature on the website. I am unsure whether I should still be copying these SimpleLogin emails that are showing as the sender and then generating a new email address to send to those.

**Comment by nguyenkims** [comment]
Do you know if the "sender email" you referred to have the `reply+` or `ra+` prefix? In this case they are the reverse-alias that allows you to send emails from your alias without revealing your original mailbox.
